.gitignore

```diff
@@ -16,6 +16,7 @@ artifacts.zip
 .node-version
 .sbt
 .sbt.*
+package-lock.json
```

Comment:
Not sure @SiAdcock and team will agree with that.

Reply:
I agree with @regiskuckaertz, in that I don't agree with this 😄 

We shouldn't be generating a `package-lock.json`, since we should never be running `npm install`. And if we do by mistake, seeing a `package-lock.json` in unstaged changes (or in a PR!) is a nice indicator that we forgot to use `yarn`

Reply:
Ok thanks. I'll revert that change ⏪